UkolPlanovac/Event.cs

```diff
@@ -1,50 +1,47 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
 using System.Globalization;
 
 namespace UkolPlanovac
 {
     public class Event
     {
-        private String EventName;
+        private string EventName;
         private DateTime EventDate;
 
         private static List<Event> EventList = [];
         private static Dictionary<DateTime, int> EventDict = [];
-        
+
         public Event(string name, DateTime date)
         {
             EventName = name;
             EventDate = date;
         }
 
-        public static (bool, string, DateTime) CheckInput(string input)
+        public static (bool IsValid, string Name, DateTime Date) ParseEvent(string input)
         {
-            String[] inputSplitted = input.Split(";");
+            string[] inputSplitted = input.Split(";");
             DateTime date = DateTime.Now;
-            bool result = false;
-
-            if ((inputSplitted.Length == 3) && (inputSplitted[0] == "EVENT") && DateTime.TryParseExact(inputSplitted[2], "yyyy-MM-dd", CultureInfo.InvariantCulture, DateTimeStyles.None, out date))
-            {
-               result = true;                      
-            }
+            bool result = (
+                (inputSplitted.Length == 3) &&
+                (inputSplitted[0] == "EVENT") &&
+                DateTime.TryParseExact(
+                    inputSplitted[2],
+                    "yyyy-MM-dd",
+                    CultureInfo.InvariantCulture,
+                    DateTimeStyles.None,
+                    out date
+                )
+            );
```

Comment:
Pokud mame delsi radek (ktery se uz treba ani nevejde na obrazovku), tak je dobrym zvykem ho rozdelit na nekolik kratsich radku pro zlepseni citelnosti. Je nekolik zpusobu, jak to rozdelit, tak pokud be se ti muj nelibil, muzes to udelat po svem 🙂